### PR TITLE
* Toasts, documentation etc

### PR DESCRIPTION
**Pull request by Wagnerp**
* Bit of reworking

### CODE REVIEW

File: Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs [hunk 1]
```diff
@@ -0,0 +1,248 @@
+﻿using System;
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs:1]: Needs License Header

**Review by Smurf-IV — CHANGES_REQUESTED** [review]
Same comment for **all** new files

File: Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs (hunk 1 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs]: Add licenses, except for documentation

File: Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs [hunk 2]
```diff
@@ -1,4 +1,33 @@
-﻿using System;
+﻿#region MIT License
+
+/*
+ * MIT License
+ *
+ * Copyright (c) 2017 - 2024 Krypton Suite
```
Comment (Smurf-IV) [inline Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs:6]: Wrong Start date for these files ?

**Review by Smurf-IV — CHANGES_REQUESTED** [review]

File: Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs (hunk 2 above)
Comment (Wagnerp) [reply Source/Krypton Toolkit/Examples/BasicToastNotificationTest.cs:6]: Corrected years

**Pull request merged**